backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -58,11 +58,6 @@ public class JiraService {
 	public static final List<String> FIELDS_IGNORE = List.of("summary", "description", "attachment", "duedate",
 			"issuelinks");
 
-	@PreDestroy
-	public void shutdownExecutor() {
-		taskExecutor.shutdown();
-	}
-
```

Comment:
I am curious about why delete these codes. Don't we need to shutdown executor before all beans destroy?